app/code/Magento/Theme/view/frontend/web/js/cookie-status.js

```diff
@@ -0,0 +1,36 @@
+define([
+    'jquery',
+    'mage/translate',
+    'Magento_Ui/js/modal/modal'
+], function($, $tr, modal){
+    'use strict';
+
+    $.widget('mage.cookieStatus', {
+        /**
+         * Init object
+         * @private
+         */
+        _init: function () {
+
+            if(!navigator.cookieEnabled) {
+                const options = {
```

Comment:
Could you please move those options to widget's `options` field so people are able to customize them without overwriting the whole method?